### PR TITLE
SDKS-787

### PR DESCRIPTION
**Pull request by witrisna**
Close response body for HTTP call (reference https://square.github.io/okhttp/interceptors/, https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response-body/)

### CODE REVIEW

**Review by spetrov — APPROVED** [review]
Looks good! Tested the fixes and they work great!

File: forgerock-auth/src/main/java/org/forgerock/android/auth/ResponseHandler.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 package org.forgerock.android.auth;
 
 import okhttp3.Response;
+
```
Comment (rodrigoareis) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/ResponseHandler.java:11]: Update year on the header

File: forgerock-auth/src/main/java/org/forgerock/android/auth/UserService.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public void onFailure(@NotNull Call call, @NotNull IOException e) {
             public void onResponse(@NotNull Call call, @NotNull Response response) {
                 if (response.isSuccessful()) {
                     try {
-                        JSONObject jsonObject = new JSONObject(Objects.requireNonNull(response.body()).string());
+                        JSONObject jsonObject = new JSONObject(response.body().string());
```
Comment (rodrigoareis) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/UserService.java:69]: Update year on the header

**Review by rodrigoareis — COMMENTED** [review]
Changes looks good, just need to update the header on two files

**Review by rodrigoareis — APPROVED** [review]
Looks good to me

**Pull request merged**